### DISCUSSION TITLE
Adjust timing without doing OCR

### DISCUSSION THREAD

**Discussion by behrisch** [SubtitleEdit/subtitleedit#8763] [Q&A]
Hi, I have a matroska file with embedded DVD subtitles (as images) and I wonder if it is possible to adjust the timing of the subtitles without going through the full OCR pipeline. It seems the preview only shows the text subtitles or is there also a way to show the original images?

**Comment by niksedk** [comment]
You can save the DVD subtitles as Blu-ray sup ... and then use File - Import - Blu-ray sup subtitle for edit...

-----------

Or...

In the OCR window you can also export to VobSub;
![image](https://github.com/user-attachments/assets/f11253fc-5071-42e2-8dc9-9db286777fc7)

And adjust timings:
![image](https://github.com/user-attachments/assets/eda0a5ba-3668-432f-a5e7-c7558ae3c5c2)

**Comment by behrisch** [comment]
Thanks for the fast answer! Maybe I was not specific enough. Is there a way to adjust timings _visually_?
